low-level/files/opt/seagate/sspl/bin/consuldump.py

```diff
@@ -42,8 +42,14 @@ def __init__(self, localtion=os.getcwd(), dir_prefix="", existing=False, name=No
                 print(f'Error[{connerr}] consul connection refused Retry Index {retry_index}')
                 time.sleep(WAIT_BEFORE_RETRY)
             except Exception as gerr:
-                print(f'Error[{gerr}] consul error')
-                break
+                # TODO: optimize the if-else here and wherever this similar code is used
```

Comment:
please create a task ticket with this PR link & TODO mentioned. A function taking care of these consul error specific check would be sufficient & then calling it from everywhere. 

Also CONSUL_ERR_STRING name is too generic, make it self explanable, like CONSUL_ERR_NO_CLUSTER_LEADER. Keep this too in ticket

Reply:
https://jts.seagate.com/browse/EOS-14757 has been created